docs/pipelines/tinycap.md

```diff
@@ -72,6 +72,32 @@ One can specify other options such as: `bowtie` (alignment) options, clipping an
 }
 ```
 
+## Taxonomy extraction 
```

Comment:
I see a lot of duplication here. Maybe better to make a separated page and link to that from all those pages?